analyzer/src/main/groovy/com/novoda/sqlite/generator/GetFieldGenerator.groovy

```diff
@@ -6,8 +6,11 @@ import groovy.text.GStringTemplateEngine
 class GetFieldGenerator {
 
     private static final String TEMPLATE = '''\
+/**
+* derived from sqlite row: name=$rowName affinity=$affinity
```

Comment:
And can we have a proper sentence?

 `Derived from the sqlite row name=$rowName affinity=$affinity.`
better:
 `Generated from the sqlite row named $rowName with an affinity of $affinity.`